nmrfx-core/src/main/java/org/nmrfx/math/VecBase.java

```diff
@@ -131,7 +131,7 @@ public VecBase(int size, boolean complex) {
     public VecBase(int size, boolean complex, PyType type) {
         super(type);
         this.isComplex = complex;
-        useApache = true;
+        useApache = complex;
```

Comment:
Changed this value here because it was set to true even when complex is false which lead to an exception in vecMat.getBytes since it tried to access cvec in a non complex vector that only had rvec set. Not sure if this is the best way to solve this?

Reply:
Undid this change, see comment about cvec null check